ocs_ci/templates/mcg/nsfs_restore_xattr.py

```
@@ -0,0 +1,30 @@
+found = []
+all_files = []
+for root, dirs, files in os.walk(base):
+    for f in files:
+        path = os.path.join(root, f)
+        try:
+            xattrs = os.listxattr(path)
+            all_files.append((path, xattrs))
+            if "user.noobaa.restore.request" in xattrs:
+                found.append(path)
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)